downstream/modules/platform/ref-controller-automation-analytics.adoc

```diff
@@ -0,0 +1,55 @@
+[id="ref-controller-automation-analytics"]
+
+= Automation Analytics
+
+When you imported your license for the first time, you were given options related to the collection of data that powers {Analytics}, a cloud service that is part of the {PlatformNameShort} subscription. 
+
+[IMPORTANT]
+====
+For opt-in of {Analytics} to have any effect, your instance of {ControllerName} must be running on {RHEL}.
+====
+
+As with Red Hat Insights, {Analytics} is built to collect the minimum amount of data needed. 
+No credential secrets, personal data, automation variables, or task output is gathered. 
+
+For more information, see xref:ref-controller-data-collection-details[Details of data collection].
+
+To enable this feature, turn on data collection for {Analytics} and enter your Red Hat customer credentials in the *Miscellaneous System settings* of the System configuration list of options in the *Settings* menu.
+
+image:configure-controller-system-misc-analytics.png[Miscellaneous System Settings]
+
+You can view the location to which the collection of insights data will be uploaded in the *{Analytics} upload URL* field on the *Details* page.
```

Comment:
You can view the location to which the collection of insights data is uploaded...